nodes/Gyazo/GyazoDescription.ts

```diff
@@ -66,6 +66,18 @@ export const gyazoOperations: INodeProperties[] = [
 					},
 				},
 			},
+			{
+				name: 'Update',
+				value: 'update',
+				description: 'Update image description and alt text',
+				action: 'Update an image',
```

Comment:
- 🔺 Update an image: image itself not updated
- 🔺 Update a capture: no other usages of "capture" on other actions
- ⭕️ Update an image attributes: makes sense
- 🔺 Update an image properties: not image properties itself

Here is my suggestion:
```suggestion
				action: 'Update an image attributes',
```